src/components/ImageView/index.js

```diff
@@ -48,12 +48,12 @@ class ImageView extends PureComponent {
     }
 
     componentDidMount() {
-        if (this.canUseTouchScreen) {
-            return;
-        }
         Image.getSize(this.props.url, (width, height) => {
```

Comment:
This change is fine but what is the use of running `setImageRegion` when a lot of its code is not used at all. 

See that in render when `if (this.canUseTouchScreen) {`  a lot of code of `setImageRegion` is useless to execute.

```js
    this.setState({
            imgWidth: width,
            zoomScale: newZoomScale,
            imgHeight: height,
            imageLeft: imgLeft,
            imageTop: imgTop,
            imageRight: imgRight,
            imageBottom: imgBottom,
        });
 ```

Reply:
This changes not necessary no more thanks you 🙏 . https://github.com/Expensify/App/pull/8761#discussion_r858771539